src/rust-crypto/backup.ts

```diff
@@ -460,7 +460,7 @@ export class RustBackupDecryptor implements BackupDecryptor {
         for (const [sessionId, sessionData] of Object.entries(ciphertexts)) {
             try {
                 const decrypted = JSON.parse(
-                    await this.decryptionKey.decryptV1(
+                    this.decryptionKey.decryptV1(
```

Comment:
this is slightly separate, but a cleanup while I was in the area.

`decryptionKey.decryptV1` doesn't return a promise.

Reply:
Yeah I have seen it now that I use WebStorm. I am wondering if you shouldn't update the bindings to make it async?

Reply:
Added an `await immediate()` in https://github.com/matrix-org/matrix-js-sdk/pull/3826/commits/131c52433043280556c1c0370be8b814da2cb4ca to make sure we don't block the event loop.